extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -63,16 +66,40 @@ export const RTC_ERROR_ENUM = {
 /**
  * @param {string} error
  * @param {string} callout
+ * @param {!Window} win
  * @param {number=} opt_rtcTime
  * @return {!Promise<!rtcResponseDef>}
  * @private
  */
-function buildErrorResponse_(error, callout, opt_rtcTime) {
+function buildErrorResponse_(
+  error, callout, errorReportingUrl, win, ampDoc, opt_rtcTime) {
   dev().warn(TAG, `RTC callout to ${callout} caused ${error}`);
+  if (errorReportingUrl) {
+    sendErrorMessage(error, errorReportingUrl, win, ampDoc);
+  }
   return Promise.resolve(/**@type {rtcResponseDef} */(
     {error, callout, rtcTime: opt_rtcTime || 0}));
 }
 
+/**
+ * @param {string} errorType Uses the RTC_ERROR_ENUM above.
+ * @param {string} errorReportingUrl
+ * @param {!Window} win
+ */
+function sendErrorMessage(errorType, errorReportingUrl, win, ampDoc) {
+  if (Math.random() < RTC_ERROR_REPORTING_FREQUENCY) {
+    const whitelist = {ERROR_TYPE: true, HREF: true};
+    const macros = {
+      ERROR_TYPE: errorType,
+      HREF: win.location.href
+    };
+    Services.urlReplacementsForDoc(ampDoc).expandUrlAsync(
+        errorReportingUrl, macros, whitelist).then(url => {
+      Services.xhrFor(win).fetch(url);
```

Comment:
Do we want to send as an XHR?  I was thinking we would use an image (see https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/transport.js#L66)